### PR TITLE
feat(route): 添加南京大学本科生交换生系统

### PR DESCRIPTION
**Pull request by cqjjjzr**
## 该 PR 相关 Issue / Involved issue

（无）

## 完整路由地址 / Example for the proposed route(s)

```routes
/nju/exchangesys/proj
/nju/exchangesys/news
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note

（无）

### CODE REVIEW

File: lib/v2/nju/exchangesys.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            url: type_dict[type][1],
+        });
+        const me$ = cheerio.load(meResponse.data);
+        meParam = me$('title').attr('data-m');
+    } catch (e) {
+        // pass
+    }
+
+    let url = type_dict[type][0] + '?page=1&limit=20';
+    if (meParam) {
```
Comment (TonyRL) [inline lib/v2/nju/exchangesys.js:26]: The value of `.me` is simply the base64 encoded value of `sys.index` unless you're logged  in

Comment (cqjjjzr) [reply lib/v2/nju/exchangesys.js:26]: so we can simply hardcode the `.me` parameter?

**Comment by cqjjjzr** [comment]
本地能过，应该考虑重跑test。

File: lib/v2/nju/radar.js [hunk 2]
```diff
@@ -17,6 +17,14 @@ module.exports = {
                 target: '/nju/dafls',
             },
         ],
+        exchangesys: [
```
Comment (TonyRL) [inline lib/v2/nju/radar.js:20]:
```suggestion
        elite: [
```
should be `elite` according to `type_dict`

File: lib/v2/nju/radar.js [hunk 3]
```diff
@@ -17,6 +17,14 @@ module.exports = {
...
+            {
+                title: '本科生交换生系统',
+                docs: 'https://docs.rsshub.app/university.html#nan-jing-da-xue-ben-ke-sheng-yuan-jiao-huan-sheng-xi-tong',
+                source: ['/exchangesystem', '/'],
```
Comment (TonyRL) [inline lib/v2/nju/radar.js:24]:
```suggestion
                source: ['/exchangesystem/index/more', '/exchangesystem', '/'],
```

File: lib/v2/nju/radar.js [hunk 4]
```diff
@@ -17,6 +17,14 @@ module.exports = {
...
+                target: '/nju/exchangesys',
```
Comment (TonyRL) [inline lib/v2/nju/radar.js:25]: `/nju/exchangesys` will not match any routes as there isn't any default value for `type`

**Pull request merged**